lib/ansiblelint/__main__.py

```diff
@@ -118,6 +119,12 @@ def main() -> int:
     for match in sorted(matches):
         print(formatter.format(match, options.colored))
 
+    # If run under GitHub Actions we also want to emit output recognized by it.
+    if os.getenv('GITHUB_WORKFLOW'):
```

Comment:
This doc https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables suggests that `GITHUB_ACTIONS` var will always equal `true` in such envs. So let's rely on this property to avoid false positives in some artificial test envs:
```suggestion
    if os.getenv('GITHUB_ACTIONS') == 'true' and os.getenv('GITHUB_WORKFLOW'):
```